### ISSUE TITLE
Write and Correct MATLAB/Octave interface

### ISSUE THREAD

**Issue by brockho** [numbbo/numbbo#233]
Also the MATLAB/Octave interface should respect the new observer/logger/suite interfaces, in particular for the single-objective BBOB suite (blocked by #161 for the moment). We also need to extend the MATLAB/Octave wrapper to work with other than Mac operating systems before releasing it.

Related to #90

**Comment by brockho** [comment]
MATLAB should work now again for example experiment and SMS-EMOA example after the latest C code API change wrt benchmark suites and observers (tested on Windows and Mac). Octave support, however, is still missing entirely which is why a new issue will be opened once this one is closed.
